recipes/qutip/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+${PYTHON} setup.py install
```

Comment:
So this means that we don't need the .sh or .bat files?

I mean when install qutip normally (linux, win, mac) I just run the:
 $ python setup.py install

if the env id conf ok, then it works